client/src/app/site/pages/meetings/pages/participants/pages/participant-list/components/participant-list/participant-list.component.ts

```diff
@@ -295,7 +295,7 @@ export class ParticipantListComponent extends BaseMeetingListViewComponent<ViewU
      */
     public setPresent(viewUser: ViewUser): void {
         const isAllowed =
-            this.operator.hasPerms(Permission.userCanManage) ||
+            this.operator.hasPerms(Permission.userCanManage, Permission.userCanManagePresence) ||
```

Comment:
You could remove the first permission, since it includes the second (if you have `can_manage`, you automatically have `can_manage_presence`).